src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: 'Avenir TTC';
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  margin: 0;
```

Comment:
No need to reset margin for the body in this task

Reply:
I've fix icon's url, so now the search icon have to be displayed when input is focused.
Margin for the body is removed.